packages/contracts-bedrock/scripts/upgrades/holocene/upgrade.dockerfile

```diff
@@ -56,11 +57,14 @@ RUN forge build
 # Deliberately run the upgrade script with invalid args to trigger a build
 RUN forge script ./scripts/upgrades/holocene/DeployUpgrade.s.sol || true
 
+RUN go get github.com/ethereum-optimism/optimism/op-chain-ops/cmd/ecotone-scalar
+
```

Comment:
nit, could move this higher up to where we do the other installations

Reply:
[1f0d63b](https://github.com/ethereum-optimism/optimism/pull/13412/commits/1f0d63b9527b200df5a10926918261d2143613e1)